<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HF Fine-Tune</string>
  </property>
  <widget class="QPushButton" name="pushButton_train">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Train</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_gpu">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>GPU</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Input Model</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_evaluate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Evaluate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_output">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Ouput path</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_input">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>651</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_output">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>651</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_log">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>651</width>
     <height>441</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_inference">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Inference</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_data">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Data path</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_data">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>651</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
